### DISCUSSION TITLE
Strings hardcoded in English

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/flux#134] [Feature Requests] [answered]
Hello,

the following components have strings hardcoded in English:

combobox.blade.php ("No results")
command/items.blade.php ("No results found")
options.blade.php ("No results")

They should be using Laravel's _("Text") notation.

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by joshhanley** [comment, marked as answer]
I've just reviewed these components and they all now have the correct translation strings, so I'm going to close this.
